tests/test_mux_4to1.py

```diff
@@ -0,0 +1,29 @@
+import cocotb
+import random
+
+from cocotb.triggers import Timer
+
+MAX_ITERATIONS = 10
+
+def verification(dut):
+    if(dut.piSel.value == 0 and (dut.poI0.value == dut.piE.value)): return True
```

Comment:
Ojo con usar "and" en lugar de "&". En este caso, que comparás de a 1 bit, es lo mismo. Pero si tenés señales más grandes no va a ser lo mismo.